components/ui/vector/Remix.tsx

```diff
@@ -68,6 +68,9 @@ function SvgRemix(props: any) {
         <symbol id="remix_svg__ri-play-circle-line" viewBox="0 0 32 32">
           <path d="M16 29.333C8.636 29.333 2.667 23.364 2.667 16S8.636 2.667 16 2.667 29.333 8.636 29.333 16 23.364 29.333 16 29.333zm0-2.666c5.891 0 10.667-4.776 10.667-10.667S21.891 5.333 16 5.333 5.333 10.109 5.333 16 10.109 26.667 16 26.667zM14.163 11.22l6.505 4.336a.535.535 0 01.002.887l-.002.001-6.507 4.336a.533.533 0 01-.828-.442v-8.675a.533.533 0 01.831-.441l-.002-.001z" />
         </symbol>
+        <symbol id="remix_svg__ri-reply-fill" viewBox="0 0 32 32">
+          <path d="M11 20L1 12l10-8v5c5.523 0 10 4.477 10 10 0 .273-.01.543-.032.81C19.46 16.95 16.458 15 13 15h-2v5z" />
+        </symbol>
```

Comment:
we have an icons metadata file in `vector/icons.remixicon` that contains the full list of icons we use in the app. This helps us to update `Remix.tsx` automatically, see `vector/README.md`. So, either update it manually or import and export it in Remixicon (recommended, less error prone).

Reply:
I missed that. I copied the path from remixicon directly, still seems simpler to me :-)
Added the new icon in the export file though.